kse/src/main/resources/org/kse/gui/crypto/resources.properties

```diff
@@ -77,29 +77,29 @@ DUpgradeCryptoStrength.jlDownloadPolicyInstructions.text = <html><b>1. Download
 DUpgradeCryptoStrength.jlDropPolicyInstructions.text     = <html><b>2. Install Unlimited Strength Jurisdiction Policy</b><br><br>Browse to the Unlimited Strength Jurisdiction Policy<br>ZIP or drop it onto the icon to the right. Finally press<br>the Upgrade button.</html>
 DUpgradeCryptoStrength.jlUpgradeInstructions.text        = <html>To upgrade cryptography to unlimited strength carry out the following<br>steps:</html>
 
-DViewCertificateFingerprint.Title                        = {0} Certificate Fingerprint
+DViewCertificateFingerprint.Title                        = {0} Certificate FingerPrint
```

Comment:
Ah, that was a misunderstanding, I meant the other way round (FingerPrint -> Fingerprint). Because fingerprint is a normal English word, there should be no need to capitalize the "p".

Reply:
fixed

Reply:
![image](https://github.com/kaikramer/keystore-explorer/assets/58091322/8f1696af-3613-4002-9782-0476ee3a4e5b)